.vsts.pipelines/jobs/ci-linux.yml

```diff
@@ -195,6 +195,15 @@ jobs:
       ArtifactName: Tarball $(artifactName)
       ArtifactType: Container
 
+  # Publish prebuilt report data to blob storage.
+  - template: ../steps/publish-prebuilt-data-sh.yml
+    parameters:
+      relativeBlobPath: prebuilt-data/$(Build.SourceBranchName)/latest/$(artifactName)
+  # Also publish per-build data to a unique location for future use.
+  - template: ../steps/publish-prebuilt-data-sh.yml
+    parameters:
+      relativeBlobPath: prebuilt-data/$(Build.SourceBranchName)/rolling/$(Build.SourceVersion)/$(Build.DefinitionName)/$(Build.BuildNumber)/$(artifactName)
```

Comment:
I figure I have to include at least `Build.SourceVersion`, `Build.DefinitionName`, and `Build.BuildNumber` in here to make sure each build has a unique path, and make it reasonable to know which commit the data is for, I'm not sure about the order though. @dseefeld @crummel Does what I have make sense to you?

Reply:
I think that makes sense.  I could see swapping `DefinitionName` and `BuildNumber` too but I'm fine with this, I can't imagine we'll have too many builds for the same commit.

Reply:
Hmm, yeah, agreed the swap makes more sense. Thanks.